### PR TITLE
Issue 1137 - Rename and extend the Log Browser

### PR DESCRIPTION
**Pull request by Deathbat2190**
The new COM Archive Tool contains aditional functionalities:

- list all available parameters with the `parameter list` command
- dump all or only specified parameters to json or plain text file with `parameter get` command
- dump timestamped logs by adding the `-t` flag to a `log get` command

### CODE REVIEW

File: sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/ArchiveBrowserHelper.java [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+    }
+  }
+
+  public static void closeLocalProvider() {
+    if(localProvider != null) {
+      localProvider.close();
+    }
+  }
+
+  public static LocalOrRemoteConsumer createConsumer(String providerURI, String databaseFile) {
```
Comment (TanguySoto) [inline sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/ArchiveBrowserHelper.java:137]: Could you please make sure that at least public methods have a minimal Java doc? I know it's annoying to do but it greatly helps when we come back to the code months later :)

File: sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/ArchiveBrowserHelper.java [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Helper methods for creating the consumer and querying the archive.
+ *
+ * @author Tanguy Soto
```
Comment (TanguySoto) [inline sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/ArchiveBrowserHelper.java:57]: Put your name here. Code looks good!

File: sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/adapters/ArchiveToParametersAdapter.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+import org.ccsds.moims.mo.mc.parameter.structures.ParameterValue;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
```
Comment (TanguySoto) [inline sdk/tools/com-archive-tool/src/main/java/esa/mo/nmf/com_archive_browser/adapters/ArchiveToParametersAdapter.java:43]: Same as for public methods, can you please put a small Java doc sentence just stating what the class does? Thanks!

**Review by TanguySoto — CHANGES_REQUESTED** [review]
Just add few java doc where I stated (I didn't repeat the comment everywhere) and it looks good! I suppose you tested the new parameters commands?

**Comment by Deathbat2190** [comment]
@TanguySoto I added more javadocs as you requested and yes I tested the parameter commands.

**Review by TanguySoto — APPROVED** [review]
Nice!

**Pull request merged**